### PR TITLE
[#5383] Add property commands for table in the Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add property command for table in Gravitino CLI

### Why are the changes needed?

To expand on the set of commands the CLI has.

Fix: #5383

### Does this PR introduce _any_ user-facing change?

No, but it adds extra commands.

### How was this patch tested?

Tested locally.

### CODE REVIEW

**Comment by justinmclean** [comment]
@shaofengshi this is ready for review

File: docs/cli.md [hunk 1]
```diff
@@ -527,6 +547,30 @@ gcli tag update --tag tagA --rename newTag
 gcli tag update --tag tagA --comment "new comment"
 ```
 
+### Fileset commands
```
Comment (shaofengshi) [inline docs/cli.md:550]: This part is unrelated with this PR; is it a merge issue?

File: docs/cli.md [hunk 2]
```diff
@@ -527,6 +547,30 @@ gcli tag update --tag tagA --rename newTag
...
+
+#### Create a fileset
+
+```bash
+gcli fileset create --name hadoop.fileset.schema --fileset example --properties managed=true,location=file:/tmp/root/schema/example
```
Comment (shaofengshi) [inline docs/cli.md:555]: This seems be the old version.

File: docs/cli.md (hunk 1 above)
Comment (justinmclean) [reply docs/cli.md:550]: Yes merge had an issue I'll fix it

File: docs/cli.md (hunk 2 above)
Comment (justinmclean) [reply docs/cli.md:555]: Yes that it, let me fix it.

**Review by shaofengshi — APPROVED** [review]
LGTM
